docker/compose/README.md

```diff
@@ -5,27 +5,32 @@ custom_edit_url: https://github.com/diem/diem/edit/master/docker/compose/README.
 ---
 
 This directory contains the following compose configurations:
-* **validator-testnet**: creates a single validator test network
-* **faucet**: creates a faucet that directly connects to validator-testnet
+* **validator-testnet**: creates a single validator test network, and a faucet that directly connects to it
+* **client**: creates a client which connects to the above validator/faucet
```

Comment:
I think this client means a cli right? maybe we need name it client-cli or cli

Reply:
doc updated